pom.xml

```diff
@@ -722,6 +723,11 @@
                 <artifactId>commons-cli</artifactId>
                 <version>${version.commons-cli}</version>
             </dependency>
+            <dependency>
+                <groupId>commons-codec</groupId>
+                <artifactId>commons-codec</artifactId>
+                <version>${version.commons-codec}</version>
+            </dependency>
```

Comment:
Is this necessary? This comment makes it a bit reluctant https://github.com/wildfly/wildfly-core/pull/4133/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R800

Reply:
@darranl could you take a look ? Is elytron the one which should be bringing in common-codec ?

Reply:
@darranl @fjuma  any input ?